docs/content/sites.md

```diff
@@ -0,0 +1,145 @@
+
+### Config Bucket
+
+Special keys in your `wrangler.toml` for using sites with Worker scripts are:
+
+| Key             | Value                                                        | Example                                                      |
+| --------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| \*`bucket`      | The file path to the static assests you wish to serve in respect to the directory containing your wrangler.toml | `bucket = "./dist"`                                          |
+| `kv-namespaces` | The glue for the static assets that live on our edge to your Worker script. ` | ```kv-namespaces = [ { binding = "MY_KV", id = "e29b263ab50e42ce9b637fa8370175e8" }]``` |
+|                 | \*`binding` is an arbitrary value that the [npm package](TODO) references when serving files from the Worker script |                                                              |
+|                 | `id` the ID of the namespace where the files live            |                                                              |
+
+\* Note you shouldn't need to touch these values.
+
+With all the appropriate files setup, you'll want to add your `wrangler.toml` the namespace ID as well as the deploy configuration.
+
+##### Add namespace ID
+
+First create a namespace
+
+```
+$ wrangler kv create STATIC_ASSETS --env=prod
+=> { "id": "c417513a199c4289a1a0c8385c58a0e7", "title": "workers-site-STATIC_ASSETS" }
+```
+
+Note [`—-env`](./environments) is optional.
+
+Add this namespace ID to your `wrangler.toml`:
+
+```
+name = "my-blog"
+type = "webpack"
+account_id = "43434f8d0d9f52a395a1472b35f23439" 
+ 
+[env.prod]
+bucket = "./dist" 
+kv-namespaces = [ { binding = "MY_KV", id = "c417513a199c4289a1a0c8385c58a0e7" }]
+```
+
+If you used `wrangler init` or `wrangler generate` with the `—site` flag, then the `binding` and `assets` fields will be set. If not, then add those values.
+
+##### Deploy Config
+
+Setup the `account_id`, `zone_id`, `route`, and `workers_dev` you wish to  publish following [the project config steps](../../#-config).
+
+### Deploy
+
+Now that you have configured your project to use the static files, time to upload them and deploy the Worker. This can be done from `wrangler publish`:
+
+```console
+$ wrangler publish
+=> Writing contents of ./public to bucket c417513a199c4289a1a0c8385c58a0e7
+=> Publishing workers-sites
+=> Success! Your site is live at myblog.examples.workers.dev
+```
+
+#### Advanced: Modifying the Worker 
+
+If you used `wrangler init` or `wrangler generate` with the `—site` flag, then your Worker should already live in `worker-site/worker.js`. If not, you'll want to configure a Worker script to serve the files from KV:
+
+First install the node package:
+
+```
+npm install @cloudflare/kv-asset-handlers
```

Comment:
this will be part of the template (in the package.json)

Reply:
I put a section and sentence explaining not needed for init or generate, but let me know if you have ideas on making that clearer 
<img width="880" alt="image" src="https://user-images.githubusercontent.com/7578652/65255583-c4fe7880-dac3-11e9-9372-f66a08c03625.png">

Reply:
Or we could just include it in buckets docs

Reply:
yeah, this section feels like "how to modify an existing worker" to use this feature. in which case i think i would prescribe:

adding the worker code, then adding the `site` entry to their wrangler.toml